website/versioned_docs/version-V1/sdk-reference/react.mdx

```diff
@@ -249,6 +255,10 @@ export const FlagDetailsComponent = () => {
 
 ## Anatomy of `withConfigCatClient()`
 
+| Parameters     | Description                                                                                                                |
+| -------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `providerId`   | Optional, Identifier of `ConfigCatProvider`. [More about multiple providers](#using-multiple-providers).                   |
```

Comment:
```suggestion
| `providerId`   | Optional. Instructs the ConfigCat Client to use a specific `ConfigCatProvider` instead of the default one. Use it if you have multiple `ConfigCatProvider`s in your code. [More about multiple providers](#using-multiple-providers).                   |
```